pkg/client/delete.go

```diff
@@ -2,9 +2,15 @@ package client
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	"github.com/kyverno/chainsaw/pkg/apis/v1alpha1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	kerror "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
🤔

Reply:
Oh shit didn't saw that and didn't even git a lint issue